static/gsAdmin/components/planList.tsx

```diff
@@ -9,13 +9,13 @@ import Form from 'sentry/components/forms/form';
 import type FormModel from 'sentry/components/forms/model';
 import type {Data, OnSubmitCallback} from 'sentry/components/forms/types';
 import {space} from 'sentry/styles/space';
-import {DataCategory} from 'sentry/types/core';
+import type {DataCategory} from 'sentry/types/core';
 import type {Organization} from 'sentry/types/organization';
 import {toTitleCase} from 'sentry/utils/string/toTitleCase';
 
 import {ANNUAL} from 'getsentry/constants';
 import type {BillingConfig, Plan, Subscription} from 'getsentry/types';
-import {getPlanCategoryName} from 'getsentry/utils/dataCategory';
+import {getPlanCategoryName, isByteCategory} from 'getsentry/utils/dataCategory';
```

Comment:
Looks like this PR might be missing the definition for `isByteCategory` in `dataCategory.tsx`.